handbook/marketing/index.md

```diff
@@ -23,4 +23,39 @@ See job descriptions and responsibilities of roles on the Marketing team:
 
 ## Definitions
 
-See "[Marketing definitions](https://docs.google.com/document/d/1h6xQeGHpMrsUUJ63Q6KupUNzq1iOpZfZSMC6ta3pLiY/edit#heading=h.pt4ztk7qhub0)".
\ No newline at end of file
+### Visitor
+
+A visitor is an anonymous individual from a targeted country has visited www.about.sourcegraph.com.
+
+### Inquiry
+
+An inquiry is an individual who has given us their email with the intention of consuming content or has given us their email at a conference or event. 
+
+### Follower
+
+A follower is an individual who has subscribed to our newsletter or followed us on social media, with the implied consent to receive future communications from us. 
+
+### Open Source User
+
+An open source user is an individual who has created a Sourcegraph account while using the Sourcegraph.com instance. 
```

Comment:
This is confusing because "Open Source User" could also mean a user on a self-hosted Sourcegraph instance using our open-source version.

Reply:
Agreed, we need to chat with Adam about this Eric